ompp-run-ubuntu/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Docker image to run openM++ models on latest Ubuntu LTS
+# Docker image to run openM++ models on latest Ubuntu 22.04
```

Comment:
I think the Ubuntu version change may have caused databaricks-python to fail the Trivy security scan.

Reply:
This is just changing a comment to reflect the pinned Ubuntu version. The Databricks image hasn't been built in months so it's just new CVEs which likely need to be resolved.